### PR TITLE
Update systemrequirements-zos.md for sdsf

### PR DESCRIPTION
**Pull request by Shobhajayanna**
This is to include SDSF under ZOS requirements for Zowe installation 

List the file(s) included in this PR:
systemrequirements-zos.md

### CODE REVIEW

File: docs/user-guide/systemrequirements-zos.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ Be sure your z/OS system meets the following prerequisites:
 
 - zFS volume has at least 1200 mb of free space for Zowe server components, the corresponding keystore, instance configuration files and logs, and third-party plug-ins.
 
+- SDSF
+
+  To install Zowe, there are a few commands which are issued to SDSF to know the status of the job submitted.
```
Comment (balhar-jakub) [inline docs/user-guide/systemrequirements-zos.md:30]: We should note the Workaround here as one exist

**Comment by pablocarle** [comment]
Please mark #3363 as resolved

File: docs/user-guide/systemrequirements-zos.md [hunk 2]
```diff
@@ -25,6 +25,20 @@ Be sure your z/OS system meets the following prerequisites:
 
 - zFS volume has at least 1200 mb of free space for Zowe server components, the corresponding keystore, instance configuration files and logs, and third-party plug-ins.
 
+- SDSF
+
+  During the installation of Zowe, there are a few operator commands issued to SDSF to know the status of the job submitted to ZOS. Hence it is expected to have SDSF installed on ZOS, otheriwse user can expect a below error message.
```
Comment (pablocarle) [inline docs/user-guide/systemrequirements-zos.md:30]: Please rephrase to state that it's used to interface with JES and used to send MVS commands (such as the start command for the Zowe Started Task)

File: docs/user-guide/systemrequirements-zos.md [hunk 3]
```diff
@@ -25,6 +25,20 @@ Be sure your z/OS system meets the following prerequisites:
...
+  IRX0043I Error running /Zowe/bin/utils/opercmd.rex, line 130: Routine not found
```
Comment (pablocarle) [inline docs/user-guide/systemrequirements-zos.md:31]:
```suggestion
   ```
   IRX0043I Error running /Zowe/bin/utils/opercmd.rex, line 130: Routine not found
   ```
```

Comment (pablocarle) [inline docs/user-guide/systemrequirements-zos.md:31]: you can enclose this in "```"

**Review by balhar-jakub — APPROVED** [review]

**Review by janan07 — APPROVED** [review]
Reviewed and edited by doc squad

**Pull request merged**